docs/source/linchpincli.rst

```diff
@@ -1,7 +1,7 @@
 Linchpin CLI
 ============
 
-Linchpin Command line is a unified tool that provides a consistent interface for interacting with most of the features of Linchpin. Linchpin commands line interface is availble with the "linchpin" command, and comes installed with Linchpin automatically. The linchpin command has subcommands like linchpin rise , linchpin drop etc. 
+Linchpin Command line is a unified tool that provides a consistent interface for interacting with most of the features of Linchpin. Linchpin commands line interface is availble with the "linchpin" command, and comes installed with Linchpin automatically. The linchpin command has subcommands like linchpin rise , linchpin drop etc.
```

Comment:
Should say 'Linchpin command line interface', not 'commands line'. 

The commands are now 'linchpin up' and 'linchpin destroy'. while  rise/drop do work, the cli does not show them as we're moving away from that usage.